boards/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -22,8 +22,10 @@ FAKE_VOID_FUNC(
     heartbeat_timeout_callback,
     enum HeartbeatOneHot,
     enum HeartbeatOneHot);
+FAKE_VALUE_FUNC(float, get_primary_flow_rate);
+FAKE_VALUE_FUNC(float, get_secondary_flow_rate);
 
-class FsmStateMachineTest : public testing::Test
+class FsmStateMachineTest : public FsmTest
```

Comment:
```suggestion
class FsmStateMachineTest : public testing::Test
```
FsmTest obsolete since #570